changelog_unreleased/cli/16062.md

```diff
@@ -0,0 +1 @@
+#### Add "--write" to the prettier failure output (#16062 by @mdentremont)
```

Comment:
The changelog needs more detail. Or, for small modifications like this, we may not need a changelog. Please choose the one you like.

Reply:
Sounds good, done!